tools/deployment/make_linux_package.sh

```diff
@@ -136,6 +137,15 @@ function main() {
     cp $INITD_SRC $INSTALL_PREFIX$INITD_DST
   fi
 
+  if [[ $DISTRO = "xenial" ]]; then
```

Comment:
What if you add this check for `xenial` on line 131 then do:
```bash
SYSTEMD_SYSCONFIG_DST=$SYSTEMD_SYSCONFIG_DST_DEBIAN
```

Then add another check around line 134 and do:
```
sed -i 's/sysconfig/default/g' $INSTALL_PREFIX$SYSTEMD_SERVICE_DST
```

would that capture all the changes? It's 2 additional if statements, which is lame, but only 2 effective statements-- better than having to clean up the RHEL-based config.